test/jasmine-junit-reporter.spec.js

```diff
@@ -0,0 +1,9 @@
+﻿/*jshint jasmine: true, node: true */
+'use strict';
+
+const jasmineReporters = require('jasmine-reporters');
+
+jasmine.getEnv().addReporter(
```

Comment:
Will this execute when I run `skyux test` from within my SPA's repo? I don't think it will. Is there some magic way we can get the same effect to happen?

If not, then these changes will result in `skyux test` creating a Cobertura file by default for SPAs. SPAs that also want a JUnit test report can add `jasmine-reporters` to their `devDependences` and can add a file like this one to their Jasmine tests.